### PR TITLE
Fix duplicated code line in docs of JavaFileUpload

### PR DESCRIPTION
**Pull request by onilton**
## Pull Request Checklist

* [X] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [X] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [X] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [X] Have you updated the documentation for both Scala and Java sections?

# Fixes

Fixes duplicated line in code for testSyncUpload in JavaFileUpload documentation.

https://www.playframework.com/documentation/2.8.x/JavaFileUpload#Testing-the-file-upload

docscreenshot

### CODE REVIEW

File: documentation/manual/working/javaGuide/main/upload/code/JavaFileUploadTest.java [hunk 1]
```diff
@@ -44,8 +44,6 @@ public void testFileUpload() throws IOException {
             FileIO.fromPath(file.toPath()),
             Files.size(file.toPath()));
 
-    // ###replace:     Http.RequestBuilder request =
-    // Helpers.fakeRequest().uri(routes.MyController.upload().url())
```
Comment (mkurz) [inline documentation/manual/working/javaGuide/main/upload/code/JavaFileUploadTest.java:48]: Actually, this is not correct.
This should be one line:
```java
        //###replace:     Http.RequestBuilder request = Helpers.fakeRequest().uri(routes.MyController.upload().url())
```
The problem is that commit 282cb7905f3b4a1b4dc97b2fbc8e8eca9224158b reformatted this one-liner and split it into two lines...

Comment (mkurz) [reply documentation/manual/working/javaGuide/main/upload/code/JavaFileUploadTest.java:48]: @onilton What really needs to be done is to `//###replace` the line:
```java
.uri("/upload")
```
with

```java
.uri(routes.MyController.upload().url())
```

Comment (onilton) [reply documentation/manual/working/javaGuide/main/upload/code/JavaFileUploadTest.java:48]: Nice! I didn't know.
Fixed in d758ed5

**Review by mkurz — APPROVED** [review]
Thanks!

**Comment by mkurz** [comment]
@Mergifyio backport 2.8.x

**Pull request merged**